docs/egress.md

```diff
@@ -41,6 +41,11 @@ You may be interested in using this capability if any of the following apply:
 - You want to force outgoing external connections to leave the cluster via
   certain Nodes, for security controls, or due to network topology restrictions.
 
+To achieve the desired egress behavior, a dummy interface that is always down is
+automatically created on the Node hosting the egress IP. This dummy interface
+facilitates the Source Network Address Translation (SNAT) and ensures that the
```

Comment:
I don't think it really facilitates SNAT. It's more to ensure that the system can reply to ARP requests for the Egress IP, assuming `arp_ignore == 0`. cc @tnqn

Reply:
The whole idea is to let user know that dummy interface can remain in a down state because its primary function is to respond to ARP/NDP requests. It does not need to handle actual data traffic.
Here is the post routing rule for egress ip, it does not contain egress interface, as righty pointed , SNAT is ruled out by use of egress interface. Will reframe the doc.
`-A ANTREA-POSTROUTING ! -o antrea-gw0 -m comment --comment "Antrea: SNAT Pod to external packets" -m mark --mark 0x1/0xff -j SNAT --to-source 172.18.0.10
`

Reply:
Not only ARP requests, also to ensure the linux system accepts tunnel traffic to this IP. I don't feel necessary to give such low-level implementation details to user doc. Can we just say the interface is supposed to be down and egress traffic will not flow though it but the interface determined by the route table?

Reply:
we could add it as a note then "A dummy interface is automatically created on the Node hosting the egress IP, the interface is intended to be down and egress traffic will not flow though but the interface determined by the route table."